app/views/layouts/_topbar.html.erb

```diff
@@ -66,15 +66,17 @@
                     <% if can_edit_my_adopters? || can_edit_all_adopters? %>
                       <li><%= link_to "Applications",  :controller => "adopters", :status => "active" %></li>
                     <% end %>
-                    <li><%= link_to "DNA List", :controller => "banned_adopters" %></li>
-                    <li><%= link_to "Manage Dogs", :controller => "dogs", :status => "active" %></li>
+                    <% if current_user.active? || current_user.admin? %>
```

Comment:
Left this comment on the PR, but this might be redundant - are there any cases where an `admin` would not be `active`?

Reply:
Not critical, but possible, was planning on addressing in a PR for refactoring Groups/Permissions.